### ISSUE TITLE
Installing form-data-parser with npm does not work

### ISSUE THREAD

**Issue by bpetetot** [mjackson/remix-the-web#34]
### Description

When I try to install `@mjackson/form-data-parser` with `npm` it does not work.

```
npm error code EUNSUPPORTEDPROTOCOL
npm error Unsupported URL Type "workspace:": workspace:^
```

### Steps to reproduce

- Node version: v22.11.0
- Npm version: 10.9.0

```
mkdir my-project
cd my-project
npm init
npm install @mjackson/form-data-parser
```

### Additional note

It works well with the 0.4.0 version:
```
npm install @mjackson/form-data-parser@0.4.0
```

Thanks for all your work!

**Comment by herkulano** [comment]
I'm having the same issue with pnpm:
```
 ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@mjackson/headers@workspace:^" is in the dependencies but no package named "@mjackson/headers" is present in the workspace

This error happened while installing the dependencies of @mjackson/form-data-parser@0.5.0
 at @mjackson/multipart-parser@0.7.1
 ```

**Comment by ryanflorence** [comment]
You can workaround this with pnpm overrides (and probably npm too, but I'm using pnpm)

```json
  "pnpm": {
    "overrides": {
      "@mjackson/multipart-parser": "0.7.1",
      "@mjackson/headers": "0.8.0"
    }
  },
```

**Comment by mjackson** [comment]
Sorry about the bad publish, it should be fixed in v0.5.1.

https://github.com/mjackson/remix-the-web/releases/tag/form-data-parser%400.5.1

**Comment by simonjodet** [comment]
Hello @mjackson , 

I have the same issue with v0.7.0. v0.6.0 can be installed with npm.
